### ISSUE TITLE
Prompt after :begin

### ISSUE THREAD

**Issue by galenwp** [urbit/archaeology#142]
After the :begin process completes and getting a “request approved” I always get dropped in to my submarine prompt. Only after pressing enter in that prompt do I get a "[waiting..]" while the hoons get pulled down. 

Perhaps the pulling is happening in the background or something, but I think we probably want to give better user feedback here. I understand it's a bit difficult to fix and falls under "%batz muck", but figured it was better to file it rather than forget it.

**Comment by ghost** [comment]
Duplicate of #8.

**Comment by galenwp** [comment]
True. Thanks.
